### ISSUE TITLE
Entering duration as +hhmm

### ISSUE THREAD

**Issue by kuzzmi** [lfos/calcurse#125]
I find it a bit confusing that `hhmm` is supported as absolute time, but `+hhmm` can't be used as duration.

I'll be checking how to add that format myself, but I'm wondering if anybody else has any issues with that.

**Comment by lhca** [comment]
Hmm. There are too many formats already (not to mention the undocumented ones). What _I_ find confusing is `+hh:mm` (and `+hhmm` is just an abbreviation for that), because `hh:mm` is the usual 24-hour clock format, but is interpreted differently in the duration context.

**Comment by lfos** [comment]
How is it interpreted differently?

**Comment by lhca** [comment]
As a time span, not as a point in time: `+12:00` means twelve hours, `12:00` means midday, `+48:00` means 48 hours, `48:00` is meaningless.

**Comment by lfos** [comment]
Well, `2:30` is the natural way humans express both the time span "two hours and thirty minutes" and the time "half past two". So it makes sense to do the same in interactive mode with an additional marker to distinguish between the two.

If you want a more theoretical argument: you can always interpret point in times as time spans since midnight.

That being said, I am not sure what to do with the `hhmm` format. It's not very natural and probably causes more trouble than it helps. Should we get rid of it?

**Comment by kuzzmi** [comment]
I agree that `hhmm` format is *not* natural. However, we're talking about data entry, and `hhmm` has one keystroke less than `hh:mm`.

**Comment by loweryaustin** [comment]
I personally prefer the hhmm format. Pressing shift and a key in between numbers is a slight inconvenience. I like calcurse for the idea of having a fluid interface that I can interact with via the keyboard only. Small things like eliminating unnecesary characters and keystrokes is what makes it an appealing interface for me.
